data_structures/binary_trees/threaded_binary_trees.c

```diff
@@ -0,0 +1,243 @@
+/**
+ * @file
+ * \brief Threaded Binary Tree is a binary tree variant in which all left child
```

Comment:
👍 on making the fixes

please keep the `\brief` a simple one line and the details separated by a blank like as such:
```c
/**
 * <other tags if any >
 * \brief a one line info of this program
 *
 * a much more elaborate description
 * goes here
 * and 
 * can span many lines
 * <other tags if any>
 */
```